locales/en-US/strings.properties

```diff
@@ -5,25 +5,35 @@ bookmarked=Bookmarked
 synced=Synced from another device
 open=Open
 topic=Topic
+# LOCALIZATION NOTE: 'bookmark' is an action meant as 'add to bookmarks'
 bookmark=Bookmark
 remove_bookmark=Remove Bookmark
 share=Share
 copy_address=Copy Address
-email_link=Email Link...
+email_link=Email Link…
 open_new_window=Open in a New Window
 open_private_window=Open in a Private Window
 dismiss=Dismiss
 delete=Delete from History
 newtab=New Tab
 search_web=Search the Web
+# LOCALIZATION NOTE: {search_term} is a placeholder for what the user has inputed
+# in the search input field: 'Search for ' + search_term + 'with'. The search engine
+# name is displayed as an icon and does not need a translation
+# e.g. 'Search for abc with'
 search_for=Search for {search_term} with
 search_settings=Change Search Settings
+# LOCALIZATION NOTE: {search_engine_name} is a placeholder for their current search
+# engine name: search_engine_name + ' Search'
+# e.g. 'Google Search'
 search_header={search_engine_name} Search
-loading=Loading...
+loading=Loading…
 welcome_title=Welcome to new tab
 welcome_body=Firefox will use this space to show your most relevant bookmarks, articles, videos, and pages you've recently visited, so you can get back to them easily.
```

Comment:
The apostrophe for `you're` are usually with `’`, e.g., https://dxr.mozilla.org/mozilla-central/source/browser/locales/en-US/chrome/browser/browser.properties#24